airbyte-integrations/bases/base-java/src/main/java/io/airbyte/integrations/base/ssh/SshTunnel.java

```diff
@@ -6,6 +6,7 @@
 
 import com.fasterxml.jackson.databind.JsonNode;
 import com.google.common.base.Preconditions;
+import com.trilead.ssh2.crypto.PEMDecoder;
```

Comment:
I searched bouncycastle code and it looks like it should support OpenSSH private keys. Did you try updating to a newer version of bouncy castle (e.g. 1.66 from current 1.66) instead of using a new library?